study-builder/studies/osteo/study.conf

```diff
@@ -2189,6 +2189,83 @@
       "dispatchToHousekeeping": true,
       "order": 1
     },
+    ## age-up next steps email
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    ## age-up re-consent/next steps reminder emails
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps1wkReminder},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 604800, # one week in seconds
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps2wkReminder},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 1209600, # two weeks in seconds
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    {
+      "trigger": {
+        "type": "REACHED_AOM"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentNextSteps4wkReminder},
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """user.studies["CMI-OSTEO"].forms["CONSENT"].isStatus("COMPLETE")""",
+      "delaySeconds": 2419200, # four weeks in seconds
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+    ## age-up re-consent complete thank-you
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "CONSENT",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "emailTemplate": ${emails.reconsentThankYou},
+        "linkedActivityCode": "CONSENT",
+        "language": "en",
+        "pdfAttachments": []
+      },
+      "dispatchToHousekeeping": true,
```

Comment:
We should have a cancel expression so this doesn't run for regular adult participants.